### ISSUE TITLE
Godot Docs 3.0 Step by Step Scripting

### ISSUE THREAD

**Issue by NobleValerian** [godotengine/godot-docs#1496]
I'm attempting to follow the documentation, but it becomes inconsistent with itself.  Goes as follows:

Type this:
```
func _ready():
    get_node("Button")
```

Type this:
```
func _on_button_pressed():
    get_node("Label").text = "HELLO!"
```

Add this:
`.connect("pressed", self, "_on_button_pressed")`
To this:
```
func _ready():
    get_node("Button")
```

Should look like this:
```
extends Panel
func _on_button_pressed():
    get_node("Label").text = "HELLO!"
func _ready():
    get_node("Button").connect("pressed", self, "_on_button_pressed")
```

Assuming I'm a beginner with limited experience following documentation designed for first time users, I'm inclined to review the code 600 times to see why I can follow the instructions perfectly and not get the result shown.  Then possibly waste time reorganizing my code to match even though it makes no practical difference in this instance, because your example inexplicably shows the code in a different order than it described writing it immediately before.

If you would correct the incorrectly typed script, or explain that the function order does not matter, that would be greatly appreciated.

(In case it's not clear why this might be relevant:  I just started learning from the documentation, and in this instance I only wasted a small amount of time.  However, if something so basic can be made needlessly confusing, what happens later when things become more sophisticated.  At this point, I can only expect this level of attention to detail will continue, making me wary about moving forward instead of excited.)

**Comment by mhilbrunner** [comment]
Thanks for the feedback, reordered the methods

**Comment by NobleValerian** [comment]
I'm brand spanking new to github, so you'll have to walk me through this.  If the issue is closed, but not fixed, what is the next step?  This change is not showing.

**Comment by mhilbrunner** [comment]
I changed it on the GitHub repository's `master` branch.

After a (usually short) time, this should be reflected *here*:
http://docs.godotengine.org/en/latest/getting_started/step_by_step/scripting.html#scripting-a-scene

(Note the */latest/* in the URL; this is the state of the master branch, think of it as the developer/alpha version)

Once the change is cherrypicked into the stable/3.0 branch, it will show up here:
http://docs.godotengine.org/en/3.0/getting_started/step_by_step/scripting.html#scripting-a-scene

(Notice the *3.0* instead of *latest*, you can also use *stable* instead of *3.0*)

So, if you want to have the bleeding edge, most up-to-date docs, use *latest*, but you may get some docs for stuff that changed in the development version of Godot or features that are not yet in a stable release.

The in-engine help and Google all point to the *3.0*/*stable* docs.

**TL;DR**
Yes, it will take some time for it to show up, if you don't want to wait, you can cheat and have a look by using the *latest* docs instead of *3.0* :)

**Comment by NobleValerian** [comment]
Thank you.
